### PR TITLE
[CI/CD] Adding pipeline for building docker image

### PR DESCRIPTION
**Pull request by AugustasV**
Adding GitHub action to build code into docker image, without pushing into docker registry.
PR with local development tools and configuration to send to docker registry #12 
Build status: https://github.com/AugustasV/nettracer-bpf/runs/4959412540?check_suite_focus=true
@pijuszczyk

### CODE REVIEW

**Review by pijuszczyk — APPROVED** [review]

**Comment by pijuszczyk** [comment]
Looks fine to me :) You're free to merge it. Once again, thanks for your pull request!

**Comment by AugustasV** [comment]
Good to know @pijuszczyk  ! But I can't merge it by myself

This branch has no conflicts with the base branch
Only those with write access to this repository can merge pull requests.

**Comment by pijuszczyk** [comment]
Ah, alright. Sorry, it's my first time managing the pull requests here ;) Thought an approval is enough. Well, ok then, I'll merge it by myself.

**Pull request merged**
